src/phoenix/server/api/input_types/PerformanceMetricInput.py

```diff
@@ -0,0 +1,106 @@
+import inspect
+from dataclasses import field, replace
+from typing import KeysView, Optional
+
+import strawberry
+from strawberry import UNSET
+
+from phoenix.core.model_schema import Column
+from phoenix.metrics.mixins import EvaluationMetric, EvaluationMetricKeywordParameters
+from phoenix.server.api.types.PerformanceMetric import PerformanceMetric
+
+
+@strawberry.input
+class MetricParametersInput:
+    pos_label_str: Optional[str] = UNSET
+    pos_label_bool: Optional[bool] = UNSET
+    pos_label_int: Optional[int] = UNSET
+    sample_weight_column_name: Optional[str] = UNSET
+
+    parameters: strawberry.Private[EvaluationMetricKeywordParameters] = field(
```

Comment:
maybe document that this is the `internal` representation of the parameters in some way?